curry.gemspec

```diff
@@ -0,0 +1,29 @@
+lib = File.expand_path("../lib", __FILE__)
```

Comment:
The `gemspec` and the `curry/version` are pretty much the only new files added as part of this pull request, others are only moving stuff into files and requiring libraries as necessary.